<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaAprendizaje</class>
 <widget class="QWidget" name="ventanaAprendizaje">
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="botonEnviar">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>20</width>
     <height>601</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="output">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonSi">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Si</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonNo">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonRecuerdos">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Recuerdos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
